### PR TITLE
YG Answer Forms Submit Post Request

### PR DESCRIPTION
**Pull request by ygroener**
Fixed my answer form so it now submits post requests when the fields are filled in properly!
Had to add state and isolate the image urls from the image uploads to use as a parameter in the request.

I didn't really add much visual functionality but here's a gif in case it helps. Let me know if theres anything to fix! 

https://user-images.githubusercontent.com/84756768/141264258-de762076-d0af-4abc-8cad-e96ddabc3347.mp4

### CODE REVIEW

**Comment by ygroener** [comment]
[Add an Answer Modal_ Post request on submit](https://trello.com/c/0Iy3gGxs/183-add-an-answer-modal-post-request-on-submit)

File: src/components/questions/FormAnswer.jsx [hunk 1]
```diff
@@ -13,23 +14,22 @@ const AnswerForm = (props) => {
...
   // These next functions allow me to track the latest file uploaded and store it in the 'file' variable. I use this to turn the file into an image at the bottom of the form where you see an image tag.
   const [file, setFile] = useState()
 
+  // this variable will hold the array of image urls in state (used in post req)
+  const [imageURLS, addImageURL] = useState([]);
+
   const uploadFile = (e) => {
-    console.log(file)
     setFile(e.target.files[0])
+    addImageURL(imageURLS => [...imageURLS, e.target.files[0].name])
```
Comment (merosenlund) [inline src/components/questions/FormAnswer.jsx:32]: Pretty cool, nice use of the spread operator!

**Review by merosenlund — APPROVED** [review]
Looks good to me!

**Comment by merosenlund** [comment]
@ygroener It looks like there are merge conflicts that will need to be resolved. Other than that this looks good to me.

**Pull request merged**